hack/buildconfig.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
```

Comment:
should cluster-ingress-operator be just ingress-operator throughout?

Reply:
The repository name is "cluster-ingress-operator", and the image name is "openshift/origin-cluster-ingress-operator", so I think it makes sense to keep "cluster-" for the imagestream, and it makes sense for the buildconfig name to match.  Are we trying to move away from using the "cluster-" prefix in general?

Reply:
Long ago we started referring to it as "ingress-operator" (e.g. openshift-ingress-operator) and I think the only reason we didn't rename the git repo and image is because it was too much trouble. So, I don't know.